### PR TITLE
gluster: Update to support Heketi backups

### PR DESCRIPTION
**Pull request by lpabon**
Needed to update the container and the deployment to mount
the database backup secret as a volume.

### CODE REVIEW

File: pkg/storage/glusterfs/heketi_kube.go [hunk 1]
```diff
@@ -75,6 +75,7 @@ func (st *GlusterStorage) deployHeketiPod(namespace string) error {
 
 	// Deployment for Heketi
 	replicas := int32(1)
+	optional_secret := true
```
Comment (xiang90) [inline pkg/storage/glusterfs/heketi_kube.go]: OptionalSecret

File: pkg/storage/glusterfs/heketi_kube.go [hunk 2]
```diff
@@ -134,6 +135,10 @@ func (st *GlusterStorage) deployHeketiPod(namespace string) error {
 									Name:      "db",
 									MountPath: "/var/lib/heketi",
 								},
+								v1.VolumeMount{
+									Name:      "heketi-db-secret",
```
Comment (xiang90) [inline pkg/storage/glusterfs/heketi_kube.go]: make this name a variable. use the same variable at line 174

**Comment by xiang90** [comment]
lgtm

File: pkg/storage/glusterfs/heketi_kube.go (hunk 1 above)
Comment (lpabon) [inline pkg/storage/glusterfs/heketi_kube.go]: Oh! sure no problem I still have my `C`  style variables in my head.

**Pull request merged**